actions.csv

```diff
@@ -1,142 +1,162 @@
-102,2019/05/09,"https://clb.org.hk/content/senior-staff-oracle%e2%80%99s-beijing-research-centre-protest-layoffs, https://technode.com/2019/05/10/oracle-programmers-100000-recruitment/, https://www.ft.com/content/16ffdcec-749d-11e9-bbad-7c18c0ea0201",protest,"pay_and_benefits, job_security","Staff at US-owned tech giant Oracle's China Development Centre in Beijing have staged protests after the company fired 900 staff from its China team, making up 60 per cent of its research and development effort there. Employees blamed tensions between the US and China for the cuts which came in the same week as the US announced additional tariffs on Chinese goods. An employee referred to the Oracle co-founder's interview with Fox Business last October, in which Larry Ellison said that ""we [the US] have a serious competition going with China, I’m on Team USA"", and ""if we let China produce more engineers than we do, if we let China's technology companies beat our technology companies, it won't be long that our military is behind technologically also"".","beijing, china",oracle,24,"white_collar_workers, trade_wars, trump",organizejs
-103,2019/05/11,"https://www.fastcompany.com/90348304/exclusive-tech-workers-organize-protest-against-palantir-on-the-github-coding-platform, https://gist.github.com/noahzgordon/29af8c13106ddc591ccaa1ea0ce6af74",protest,ethics,"Tech workers organized by the Tech Workers Coalition are protesting the Palantir's contract with ICE on the company's GitHub projects. The plan calls for organized waves of digital protesters to post a prepared message to their GitHub boards. The protest appeals to workers to consider their role, even if indirectly, in Palantir’s work with ICE. It also encourages people to add their own personal sentiments.",usa,palantir,None,"ice, industry_solidarity, twc",organizejs
-104,2019/05/22,"https://www.vox.com/recode/2019/5/22/18635604/amazon-shareholder-meeting-2019-climate-change-proposal, https://gizmodo.com/jeff-bezos-wouldn-t-even-come-on-stage-to-listen-to-his-1834957220",protest,ethics,"More than 50 Amazon employees from Amazon Employees for Climate Justice attended the shareholder meeting, where Emily Cunningham delivered an impassioned speech to shareholders urging them to support the resolution for Amazon to adopt a strong climate plan. Jeff Be
```

Comment:
a bit conflicted on this one because it doesn't seem to say that there was organizing activity from ibm workers themselves, but rather that the union drive was done externally. Should we count that?

if so, we should add `white_collar_workers` tag